doc/reference/usb/index.rst

```diff
@@ -38,6 +38,7 @@ The following Product IDs are currently used:
 * :option:`CONFIG_USB_PID_MASS_SAMPLE`
 * :option:`CONFIG_USB_PID_TESTUSB_SAMPLE`
 * :option:`CONFIG_USB_PID_WEBUSB_SAMPLE`
+* :option:`CONFIG_USB_PID_BLE_HCI_H4_SAMPLE`
```

Comment:
Something is wrong with the commit message as this also adds the driver.

Reply:
Well it was meant to add the PID with its driver, or are you suggesting this should be splited in 2 commits?

Reply:
No, but please update the commit title, it somehow does not reflect the changes in the patch self.

Reply:
Sure, will do update the description.